MapboxMobileEvents/MMEMetrics.h

```diff
@@ -0,0 +1,27 @@
+#import <Foundation/Foundation.h>
+#import <CoreLocation/Corelocation.h>
+
+NS_ASSUME_NONNULL_BEGIN
+
+@interface MMEMetrics : NSObject
+
+@property (nonatomic) NSUInteger requests;
+@property (nonatomic) NSUInteger totalDataTransfer;
+@property (nonatomic) NSUInteger cellDataTransfer;
+@property (nonatomic) NSUInteger wifiDataTransfer;
+@property (nonatomic) NSUInteger appWakeups;
+@property (nonatomic) NSUInteger eventCountFailed;
+@property (nonatomic) NSUInteger eventCountTotal;
+@property (nonatomic) NSUInteger eventCountMax;
+@property (nonatomic) NSInteger deviceTimeDrift;
+@property (nonatomic) CLLocationDegrees deviceLat;
```

Comment:
Ideally we should replace `deviceLat` and `deviceLon` by a single `deviceCoordinate`.